src/runtime-rs/crates/runtimes/src/shim_mgmt/client.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#![allow(dead_code)]
```

Comment:
Why allow dead_code?

Reply:
I allowed dead_code because the PUT and POST method are not used. From another comment you said the PUT/POST function should be added when needed, therefore I think I can remove the #![allow(dead_code)} and the function put/post to fix this.

Reply:
The `#![allow(dead_code)` is removed now.